nilearn/regions/tests/test_hierarchical_kmeans_clustering.py

```diff
@@ -16,30 +15,25 @@
 
 extra_valid_checks = [
     "check_clusterer_compute_labels_predict",
-    "check_clustering",
```

Comment:
Had to remove some of the sklearn checks. The valid checks are more similar to ReNA clustering now